packages/terra-demographics-banner/src/selectable-demographics-banner/_SelectableTile.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './SelectableDemographicsBanner.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child Nodes that are placed within selectable content.
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: '',
+};
+
+const SelectableTile = ({
+  children,
+  ...customProps
+}) => (
+  <div {...customProps} role="button" tabIndex="0" className={cx('selectable-tile')}>
```

Comment:
Should we be setting role button? How does this affect screen readers and accessibilitity?

Reply:
role button was added to keep it align with info button and as well to meet requirements of interactive element. but it does not add any extra value for screen readers. Since we are still not sure of functionality of selectable-tiles in demographics banner I have added role as button. will update this role once we get input from UX on functionality of selectable-tiles.

Reply:
should we consider changing the div into an actual button? we'd get focus and keyboard navigation for free.

Reply:
yes we can change it to button but it requires bit of CSS tweaks to fix wrapping styles and alignment of other items inside banner. So keeping it has div for better consistency in styling.

Reply:
this is actually a topic touched on by a recent accessibility talk. a key point - we should always strive for markup that best describes function (semantic html), not appearance.

an html button conveys that a region is clickable. a div does not. i'd prefer we change this into a button to get a11y for free and update the styles, rather than converting a div into a button. i'm curious what others think.